pom.xml

```diff
@@ -137,7 +137,7 @@
         <jetty-schemas.version>3.1.M0</jetty-schemas.version>
         <airline.version>0.6</airline.version>
         <mockwebserver.version>20121111</mockwebserver.version>
-        <freemarker.version>2.3.22</freemarker.version>
+        <freemarker.version>2.3.25-incubating</freemarker.version>
```

Comment:
@bostko the freemarker project was recently moved to Apache - all such projects start as "incubating", and must therefore have "-incubating" in their release names, until they graduate as a top-level project. It was the same for Apache Brooklyn.

The "-incubating" says nothing about the quality or maturity of the product/release.